change print to retunr

taniavill opened the pull request.

tvillanueva committed:
Updated functions

lastone doesnt work

tvillanueva committed:
change print for return

tvillanueva committed:
updated

seppo0010 commented:
usá mensajes de commit más descriptivos que "updated", y en presente, por ejemplo "enable tests, accept varargs for sum"

tvillanueva committed:
try

tvillanueva committed:
try 2

fernandezpablo85 committed:
Update pythonapp.yml

tvillanueva committed:
sum_all fixed

tvillanueva committed:
Merge branch 'master' of https://github.com/taniavill/python_init

tvillanueva committed:
Update number.py

All test enable, accept vararg and list as argument

fernandezpablo85 commented on numbers/number.py:
```diff
@@ -1,14 +1,19 @@
-def sum_all(arr):
-    pass
+def add(a, b):
+    return(a+b)
```
no hacen falta los paréntesis acá 
```suggestion
    return a + b

```

fernandezpablo85 commented on numbers/number.py:
```diff
@@ -1,14 +1,19 @@
+    return(a+b)
+
+def is_even(n):
+    return(n%2 == 0)
```
Lo mismo acá

```suggestion
    return n % 2 == 0

```

fernandezpablo85 commented on numbers/number.py:
```diff
@@ -1,14 +1,19 @@
+    return(n%2 == 0)
+    
+def is_odd(n):
+    return(n%2 != 0)
```
```suggestion
    return n % 2 != 0

```

fernandezpablo85 commented on numbers/number.py:
```diff
@@ -1,14 +1,19 @@
+    return(n%2 != 0)
+
+def sum_all(*arr):
+    add_all = 0
```
a `add_all` la llamaría `total`

fernandezpablo85 commented on numbers/number.py:
```diff
@@ -1,14 +1,19 @@
+
+def sum_all(*arr):
+    add_all = 0
+    if type(arr[0]) == list:
```
la forma correcta de preguntar por un tipo en python es 

`if isinstance(arr[0], list)`

fernandezpablo85 commented on numbers/number.py:
```diff
@@ -1,14 +1,19 @@
+    if type(arr[0]) == list:
+        arr=list(*arr)
+    for numb in arr:
+            add_all += numb
```
si bien anda, no es bueno mezclar indentación. si usabas 4 espacios antes no uses 8 acá

taniavill replied:
@fernandezpablo85 no encontré otra solución. Alguna pista?

seppo0010 commented on numbers/number_test.py:
```diff
@@ -1,46 +1,47 @@
+import number
+
+
+#@pytest.mark.skip()
```
es mejor borrar las líneas en vez de comentarlas :)

tvillanueva committed:
sum_all arreglado y función de triángulo creada

mejora de number.sum_all  con funcion built in
la función tirangle.draw dibuja un triángulo pero no pasa la prueba, no encuentro el error, ayuda!

seppo0010 commented:
Me parece que lo del triangulo tendría que ir en otro PR ya.

taniavill commented:
oka, cierro este entonces y lo subo a otro?


El mié., 29 abr. 2020 a las 9:40, Sebastian Waisbrot (<
notifications@github.com>) escribió:

> Me parece que lo del triangulo tendría que ir en otro PR ya.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fernandezpablo85/python_init/pull/2#issuecomment-621177067>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APKAKVKS7FDWPMH6SBS3HCLRPAN2TANCNFSM4MSII4FQ>
> .
>
